third_party/repositories.bzl

```diff
@@ -226,10 +230,39 @@ def xchammer_dependencies():
         ]),
     )
 
+    namespaced_new_git_repository(
+        name = "SwiftCLI",
+        remote = "https://github.com/jakeheis/SwiftCLI.git",
+        commit = "5318c37d3cacc8780f50b87a8840a6774320ebdf",
+        build_file_content = namespaced_build_file([
+            namespaced_swift_library(
+                name = "SwiftCLI",
+                srcs = ["Sources/**/*.swift"],
+            ),
+        ]),
+    )
+
+    namespaced_new_git_repository(
+        name = "SwiftShell",
+        remote = "https://github.com/kareman/SwiftShell",
+        commit = "beebe43c986d89ea5359ac3adcb42dac94e5e08a",
+        build_file_content = namespaced_build_file([
+            namespaced_swift_library(
+                name = "SwiftShell",
+                srcs = ["Sources/**/*.swift"],
+                copts = [
+                    "-swift-version",
```

Comment:
Is there a reason to build with Swift 4.2? Are they not updated to Swift 5 yet?

Reply:
I've blindly set these rules up to mirror their `Package.swift`'s